tasks/main.yml

```diff
@@ -160,14 +158,15 @@
   with_items:
     - python-virtualenv
     - python-dev
-    - libxml2-dev 
+    - libxml2-dev
     - libxslt1-dev
-    
+    - libz-dev
```

Comment:
This is a virtual package; it's provided by `zlib1g-dev` in 14.04, probably also 12.04.